Define FD_SETSIZE on Cygwin regardless of XWin DDX

All DDXs segfault on Cygwin unless -DFD_SETSIZE=256 is set, so make sure
we do so whether or not we are building XWin.

Signed-off-by: Yaakov Selkowitz <yselkowitz-Rn4VEauK+AKRv+LV9MX5uipxlwaOVQ5f@public.gmane.org>
---
 configure.ac |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/configure.ac b/configure.ac
index a0bf0ab..e899b4a 100644
--- a/configure.ac
+++ b/configure.ac
@@ -424,6 +424,9 @@ case $host_os in
   darwin*)
 	AC_DEFINE(CSRG_BASED, 1, [System is BSD-like])
 	;;
+  cygwin*)
+	CFLAGS="$CFLAGS -DFD_SETSIZE=256"
+	;;
 esac
 
 dnl augment XORG_RELEASE_VERSION for our snapshot number and to expose the
@@ -1730,8 +1733,6 @@ if test "x$XWIN" = xyes; then
 			       AC_DEFINE(ROOTLESS,1,[Build Rootless code])
 			       CFLAGS="$CFLAGS -DROOTLESS_WORKAROUND"
 			fi
-
-			CFLAGS="$CFLAGS -DFD_SETSIZE=256"
 			;;
 		mingw*)
 			XWIN_SERVER_NAME=Xming
--
1.6.4.2 
